influxdb3_wal/src/object_store.rs

```diff
@@ -25,6 +36,11 @@ pub struct WalObjectStore {
     added_file_notifiers: parking_lot::Mutex<Vec<Arc<dyn WalFileNotifier>>>,
     /// Buffered wal ops go in here along with the state to track when to snapshot
     flush_buffer: Mutex<FlushBuffer>,
+    /// oldest, last and latest wal file nums are used to keep track of what wal files
+    /// to remove from the OS
+    num_wal_files_to_keep: u64,
```

Comment:
same, rename this so it indicates that it's the number of snapshotted wal files to keep.